verbose_debugging_reports.md

```diff
@@ -94,24 +111,27 @@ This table defines the fields in the `body` dictionary.
-| `source-noised` | ✓ | ❌ | ✓ | ✓ | ✓ | ❌ |
-| `source-storage-limit` | ✓ | ✓ | ✓ | ✓ | ✓ | ❌ |
-| `source-unknown-error` | ✓ | ❌ | ✓ | ✓ | ✓ | ❌ |
-| `trigger-no-matching-source` | ✓ | ❌ | ❌ | ❌ | ❌ | ✓ |
-| `trigger-no-matching-filter-data` | ✓ | ❌ | ✓ | ✓ | ✓ | ✓ |
-| `trigger-attributions-per-source-destination-limit` | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ |
-| `trigger-reporting-origin-limit` | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ |
-| `trigger-event-deduplicated` | ✓ | ❌ | ✓ | ✓ | ✓ | ✓ |
-| `trigger-event-no-matching-configurations` | ✓ | ❌ | ✓ | ✓ | ✓ | ✓ |
-| `trigger-event-noise` | ✓ | ❌ | ✓ | ✓ | ✓ | ✓ |
-| `trigger-event-storage-limit` | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ |
-| `trigger-event-report-window-passed` | ✓ | ❌ | ✓ | ✓ | ✓ | ✓ |
-| `trigger-aggregate-deduplicated` | ✓ | ❌ | ✓ | ✓ | ✓ | ✓ |
-| `trigger-aggregate-no-contributions` | ✓ | ❌ | ✓ | ✓ | ✓ | ✓ |
-| `trigger-aggregate-insufficient-budget` | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ |
-| `trigger-aggregate-storage-limit` | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ |
-| `trigger-aggregate-report-window-passed` | ✓ | ❌ | ✓ | ✓ | ✓ | ✓ |
-| `trigger-unknown-error` | ✓ | ❌ | ✓ | ✓ | ✓ | ✓ |
+| [`source-destination-limit`](#source-destination-limit) | ✓ | ✓ | ✓ | ✓ | ✓ | ❌ |
+| [`source-noised`](#source-noised-cookie-based) | ✓ | ❌ | ✓ | ✓ | ✓ | ❌ |
+| [`source-storage-limit`](#source-storage-limit-cookie-based) | ✓ | ✓ | ✓ | ✓ | ✓ | ❌ |
+| [`source-unknown-error`](#source-unknown-error-cookie-based) | ✓ | ❌ | ✓ | ✓ | ✓ | ❌ |
+| [`trigger-no-matching-source`](#trigger-no-matching-source) | ✓ | ❌ | ❌ | ❌ | ❌ | ✓ |
+| [`trigger-no-matching-filter-data`](#trigger-no-matching-filter-data) | ✓ | ❌ | ✓ | ✓ | ✓ | ✓ |
+| [`trigger-attributions-per-source-destination-limit`](#trigger-attributions-per-source-destination-limit) | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ |
+| [`trigger-reporting-origin-limit`](#trigger-reporting-origin-limit) | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ |
+| [`trigger-event-deduplicated`](#trigger-event-deduplicated) | ✓ | ❌ | ✓ | ✓ | ✓ | ✓ |
+|
+[`trigger-event-no-matching-configurations`](#trigger-event-no-matching-configurations) | ✓ | ❌ | ✓ | ✓ | ✓ | ✓ |
+| [`trigger-event-noise`](#trigger-event-noise) | ✓ | ❌ | ✓ | ✓ | ✓ | ✓ |
+| [`trigger-event-storage-limit`](#trigger-event-storage-limit) | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ |
+| [`trigger-event-report-window-passed`](#trigger-event-report-window-passed) | ✓ | ❌ | ✓ | ✓ | ✓ | ✓ |
+| [`trigger-aggregate-deduplicated`](#trigger-aggregate-deduplicated) | ✓ | ❌ | ✓ | ✓ | ✓ | ✓ |
+| [`trigger-aggregate-no-contributions`](#trigger-aggregate-no-contributions) | ✓ | ❌ | ✓ | ✓ | ✓ | ✓ |
+|
+[`trigger-aggregate-insufficient-budget`](#trigger-aggregate-insufficient-budget) | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ |
+| [`trigger-aggregate-storage-limit`](#trigger-aggregate-storage-limit) | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ |
+|
+[`trigger-aggregate-report-window-passed`](#trigger-aggregate-report-window-passed) | ✓ | ❌ | ✓ | ✓ | ✓ | ✓ |
+| [`trigger-unknown-error`](#trigger-unknown-error) | ✓ | ❌ | ✓ | ✓ | ✓ | ✓ |
```

Comment:
Something is wrong with the formatting here, if you look at the rendered file.

Reply:
Fixed, thanks.